core/db_classes/EE_Registration.class.php

```diff
@@ -2082,4 +2104,24 @@ public function get_related_primary_datetime()
         );
         return $this->event()->primary_datetime();
     }
+
+    public function name()
+    {
+        $attendee = $this->attendee();
+        if ($attendee instanceof EE_Attendee) {
+            $attendee_name = $attendee->full_name();
+        } else {
+            $attendee_name = esc_html__('Unknown', 'event_espresso');
+        }
+        return esc_html(
+            sprintf(
+                // translators: 1: registration code, 2: number, 3: group size, 4: attendee name.
+                __('Registration with code "%1$s", %2$d of %3$d for %4$s', 'event_espresso'),
+                $this->reg_code(),
+                $this->count(),
+                $this->group_size(),
+                $attendee_name
+            )
+        );
+    }
```

Comment:
Where is this used?
If this is totally new (looks it) then maybe we can come up with a better name for the method than just `name` ?

It certainly appears to be providing more info than just the registrant's name.

how about:

- description
- regDescription
- registrationDescription
- something else ?

i obviously couldn't come up with something more descriptive and/or appropriate than something related to "description"

Reply:
It's actually overriding a method in `EE_Base_Class` which is meant to provide a pretty name we can display to users when there's dynamic code. I was using it earlier in this branch but no more. 
I've replaced this method's body to instead just return `attendeeName()` as it sounds like that's what you were expecting a method by this name to return.